README.md

```diff
@@ -172,10 +169,15 @@ use Gordalina\MixpanelBundle\Annotation as Mixpanel;
 class OrderController
 {
     /**
-     * @Mixpanel\Track("Order Completed")
-     * @Mixpanel\TrackCharge(id=@Mixpanel\Expression("user.getId()"), amount=@Mixpanel\Expression("order.getAmount()"))
+     * @Mixpanel\Track("Order Completed", props={
+     *      "user": @Mixpanel\Expression("user.getId()")
+     * })
+     * @Mixpanel\TrackCharge(
```

Comment:
Can you add an example where the we do not need to pass the `id`, so that the consumer learns about this, ideally, next to the note written below.